packages/backend/src/accounting/tigerbeetle/utils.test.ts

```diff
@@ -0,0 +1,33 @@
+import { v4 as uuid } from 'uuid'
+import { fromTigerBeetleId, toTigerBeetleId, uuidToBigInt } from './utils'
+
+describe('TigerBeetle Utils Test', (): void => {
+  test('Test TigerBeetle Utils asset type', async (): Promise<void> => {
+    const ledger = 1n
+    expect(fromTigerBeetleId(ledger)).toEqual(
+      `00000000-0000-0000-0000-00000000000${ledger}`
+    )
+  })
+
+  test('Test TigerBeetle Utils liquidity accounts', async (): Promise<void> => {
+    const uuidInitial = uuid()
+    const tbId = toTigerBeetleId(uuidInitial)
+    const fromTb = fromTigerBeetleId(tbId)
+    expect(fromTb).toEqual(`${uuidInitial}`)
+
+    // Invalid UUID:
+    try {
+      toTigerBeetleId(`${uuidInitial}-WRONG`)
+    } catch (error) {
+      expect(error).toBeInstanceOf(Error)
+      // @ts-expect-error error message will always be present:
+      expect(error.message).toEqual('wrong format of id')
+    }
+  })
+
+  test('Test ', async (): Promise<void> => {
+    expect(uuidToBigInt('1ed4bdd8-adba-4e43-a76c-278ec1ae23cc')).toEqual(
+      40981457350021189385292318948847395788n
+    )
+  })
```

Comment:
This test could pass incorrectly if `toTigerBeetleId` doesn't throw.

Could use `expect.assertions` to ensure the correct number of assertions were made. It's used around our tests for this purpose.

Reply:
Thank you @BlairCurrey . Done.